guideline-12.md

```diff
@@ -1,9 +1,20 @@
 <h4>12. Public facing pages on WordPress.org (readmes) must not spam.</h4>
 
-Public facing pages, including readmes and translation files, may not be used to spam. Spammy behavior includes (but is not limited to) unnecessary affiliate links, tags to competitors plugins, use of over 12 tags total, blackhat SEO, and keyword stuffing.
+Public facing pages, including readmes and translation files, may not be used to spam or advertise.
 
-Links to directly required products, such as themes or other plugins required for the plugin's use, are permitted within moderation. Similarly, related products may be used in tags but not competitors. If a plugin is a WooCommerce extension, it may use the tag 'woocommerce.' However if the plugin is an alternative to Akismet, it may not use that term as a tag. Repetitive use of a tag or specific term is considered to be keyword stuffing, and is not permitted.
+Prohibited behavior includes (but is not limited to):
+<ul>
+    <li>unnecessary affiliate links</li>
+    <li>tags to competitors plugins</li>
+    <li>use of over 12 tags total</li>
+    <li>blackhat SEO practices</li>
+    <li>content spinning</li>
+    <li>advertising services</li>
+    <li>keyword stuffing</li>
+</ul>
 
-Readmes are to be written for people, not bots.
+Links to directly required products, such as themes or other plugins needed for use, are permitted within reason. Affiliate links must be disclosed and must directly link to the affiliate service, not a redirect or cloaked URL. Promoting products and services unrelated to the plugin, such as discounts to hire a company for bespoke work, are not permitted
```

Comment:
Wonder if “directly required” will be tricky to translate. Is “dependency” better or worse?

Reply:
> I wonder if this might be a good time to also swap out “prohibit” for “not allowed” to match other “not/allowed” usages in this same document?

Yeah, consistency would be good there, huh? Pushing that...